src/main/kotlin/com/kotlin/boardproject/domain/comment/domain/Comment.kt

```diff
@@ -1,9 +1,12 @@
-package com.kotlin.boardproject.model
-
-import com.kotlin.boardproject.common.enums.ErrorCode
-import com.kotlin.boardproject.common.enums.PostStatus
-import com.kotlin.boardproject.common.exception.ConditionConflictException
-import com.kotlin.boardproject.dto.comment.UpdateCommentRequestDto
+package com.kotlin.boardproject.domain.comment.domain
+
+import com.kotlin.boardproject.domain.post.domain.BasePost
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end